db/migrate/20200331111947_add_reporting_org_association_to_activity.rb

```diff
@@ -0,0 +1,37 @@
+class AddReportingOrgAssociationToActivity < ActiveRecord::Migration[6.0]
+  def up
+    add_reference :activities, :reporting_organisation, type: :uuid, foreign_key: {to_table: :organisations}
+
+    ActiveRecord::Base.transaction do
+      activities = Activity.where(reporting_organisation: nil)
```

Comment:
Given the association is created a few lines about I expect this will always be nil. I presume this is being used as a guard in case we run the migration more than once, which I'm not sure we will be doing in real environments.